client/analytics/report/revenue.js

```diff
@@ -78,34 +95,81 @@ class RevenueReport extends Component {
 				</a>
 			);
 			return [
-				row.start_date,
-				orderLink,
-				formatCurrency( gross_revenue ),
-				formatCurrency( refunds ),
-				formatCurrency( coupons ),
-				formatCurrency( taxes ),
-				formatCurrency( shipping ),
-				formatCurrency( net_revenue ),
+				{
+					display: <input type="checkbox" />,
+					value: false,
+				},
+				{
+					display: formatDate( 'm/d/Y', row.date_start ),
+					value: row.date_start,
+				},
+				{
+					display: orderLink,
+					value: Number( orders_count ),
+				},
+				{
+					display: formatCurrency( gross_revenue ),
+					value: getCurrencyFormatDecimal( gross_revenue ),
+				},
+				{
+					display: formatCurrency( refunds ),
+					value: getCurrencyFormatDecimal( refunds ),
+				},
+				{
+					display: formatCurrency( coupons ),
+					value: getCurrencyFormatDecimal( coupons ),
+				},
+				{
+					display: formatCurrency( taxes ),
+					value: getCurrencyFormatDecimal( taxes ),
+				},
+				{
+					display: formatCurrency( shipping ),
+					value: getCurrencyFormatDecimal( shipping ),
+				},
+				{
+					display: formatCurrency( net_revenue ),
+					value: getCurrencyFormatDecimal( net_revenue ),
+				},
 			];
 		} );
 	}
 
+	getSummaryContent( data = [] ) {
+		return [
+			{
+				label: __( 'gross revenue', 'wc-admin' ),
+				value: formatCurrency( data.gross_revenue ),
+			},
+			{
+				label: __( 'refunds', 'wc-admin' ),
+				value: formatCurrency( data.refunds ),
+			},
+			{
+				label: __( 'coupons', 'wc-admin' ),
+				value: formatCurrency( data.coupons ),
+			},
+			{
+				label: __( 'taxes', 'wc-admin' ),
+				value: formatCurrency( data.taxes ),
+			},
+			{
+				label: __( 'shipping', 'wc-admin' ),
+				value: formatCurrency( data.shipping ),
+			},
+			{
+				label: __( 'net revenue', 'wc-admin' ),
+				value: formatCurrency( data.net_revenue ),
+			},
+		];
+	}
+
 	render() {
 		const { path, query } = this.props;
-		const summaryStats = get( this.state.stats, 'totals', {} );
-		const intervalStats = get( this.state.stats, 'intervals', [] );
-		const rows = this.getRowsContent( intervalStats ) || [];
-
-		const headers = [
-			__( 'Date', 'wc-admin' ),
-			__( 'Orders', 'wc-admin' ),
-			__( 'Gross Revenue', 'wc-admin' ),
-			__( 'Refunds', 'wc-admin' ),
-			__( 'Coupons', 'wc-admin' ),
-			__( 'Taxes', 'wc-admin' ),
-			__( 'Shipping', 'wc-admin' ),
-			__( 'Net Revenue', 'wc-admin' ),
-		];
+		const { totals = [], intervals = {} } = this.state.stats;
```

Comment:
Likewise should it be `totals = {}`? at least lower down `totals` is being used like an object.